crates/control/src/led_status.rs

```diff
@@ -171,6 +200,38 @@ impl LedStatus {
         Ok(MainOutputs { leds: leds.into() })
     }
 
+    fn get_ears(
+        filter_whistle_detected: bool,
+        cycle_start_time: SystemTime,
+        last_game_controller_message: Option<SystemTime>,
+        blink_state: bool,
+        current_maximum_temperature: f32,
+    ) -> Ear {
+        if filter_whistle_detected {
+            return Ear::full_ears(1.0);
+        }
+
+        if last_game_controller_message.is_some_and(|timestamp| {
+            cycle_start_time
+                .duration_since(timestamp)
+                .expect("time ran backwards")
+                > Duration::from_millis(5000)
+        }) {
+            if blink_state {
+                return Ear::full_ears(1.0);
+            } else {
+                return Ear::full_ears(0.0);
+            }
+        }
+
+        let minimum_temperature = 30.0;
+
+        let relative_temperature = (current_maximum_temperature - minimum_temperature)
+            / (105.0 - minimum_temperature).floor();
```

Comment:
You named the minimum temperature constant but not the maximum. Please do so.

Reply:
Done